### ISSUE TITLE
[Patienten registrieren] Tabs statt runter skrollen

### ISSUE THREAD

**Issue by JonasCir** [ImisDevelopers/1_011_a_infektionsfall_uebermittellung#358]
Im Moment sind alle Felder untereinander bei der Patientenregistrierung, dass macht das ganze meiner Meinung nach unübersichtlich. Machen Tabs Sinn dafür?

![register](https://user-images.githubusercontent.com/20031875/82315960-6c71d900-99cc-11ea-83a8-248756a77e28.png)

**Comment by btobi** [comment]
Der Vorteil an dieser Ansicht ist, dass der User sich einfach von oben nach unten durcharbeiten kann. Bei Tabs würde man sich innerhalb des Tabs von oben nach unten durcharbeiten, dann wieder hochgehen um zum nächsten Tab zu wechseln. 

Im Fall eines Validierungsfehlers könnte es dann auch schwierig sein, den Fehler zu finden, bzw. wäre es wahrscheinlich aufwändiger in der Implementierung, das in der UI schön darzustellen. 

Vielleicht hat noch jemand anderes eine Meinung dazu.

**Comment by lordpomeroy** [comment]
Sehe ich ähnlich wie @btobi. Würde aber eine UI Überarbeitung der Seite nicht ausschließen, um effizienter zu arbeiten. Ob das dann Tabs sind oder so wie aktuell oder noch eine dritte Variante würde ich nach dem Feedback vom Gesundheitsamt mal einen UI Experten Fragen.

**Comment by btobi** [comment]
Genau, ich denke allein in der aktuellen Variante haben wir viele Änderungen mit großen Hebel für UI / UX Verbesserungen.

Ein paar Beispiele:
- Größere Gruppierungen vorvalidieren (nicht erst beim Abschicken des Formulars), also Eingaben in "Personendaten" validieren, sobald man zu "Exposition" wechselt, und das Validierungsergebnis im Tab-Header einblenden (bspw. "3 Pflichtfelder nicht ausgefüllt" o.ä.)
- Alternativ oder ergänzend: besseres Führen des Users zu den Formularfehlern, bspw. durch eine Sektion rechts vom Formular mit allen Validierungsfehlern / fehlenden Angaben mit Click-to-scroll
- [Feldbeschrifungen über dem Textfeld statt links daneben](https://ai.googleblog.com/2014/07/simple-is-better-making-your-web-forms.html)
- Sektionen klarer trennen
- Ausrichtung der Felder und deren Beschriftungen vereinheitlichen (aktuell haben wir links, rechts und zentriert ausgerichteten Text)
